c++/Makefile.am

```diff
@@ -181,12 +184,14 @@ includecapnp_HEADERS =                                         \
   src/capnp/rpc.capnp.h                                        \
   src/capnp/rpc-twoparty.capnp.h                               \
   src/capnp/persistent.capnp.h                                 \
-  src/capnp/ez-rpc.h
+  src/capnp/ez-rpc.h                                           \
+  src/capnp/compat/json.h                                      \
```

Comment:
Unfortunately automake is terrible and listing the headers here will cause them to be installed in `/usr/include/capnp` (without `compat`). You'll need to define a new variable `includecapnpcompat_HEADERS`, similar to `includekjcompat_HEADERS` above.